shopelectro/tests/tests_js_analytics.py

```diff
@@ -100,6 +99,53 @@ def get_goals(self) -> selenium.Goals:
         goals.fetch()
         return goals
 
+    def assert_add(self, product: Product, goal_position: int):  # Ignore CPDBear
+        reached_goals = self.get_goals()
+        self.assertTrue(reached_goals)
+        reached = reached_goals[goal_position]
+
+        self.assertIn('add', reached)
+        self.assertEqual(reached['currencyCode'], 'RUB')
```

Comment:
seems we should improve Goals class interface. All this dict fields is some Jedi knowledge.
Update related #940 task with the comment